src/background.js

```diff
@@ -118,24 +128,28 @@ contextMenus = Object.assign({
     click: browserCommands.openManage,
   },
 }, prefs.get('editor.contextDelete') && {
-  'editor.contextDelete': {
-    title: 'editDeleteText',
-    type: 'normal',
-    contexts: ['editable'],
-    documentUrlPatterns: [URLS.ownOrigin + 'edit*'],
-    click: (info, tab) => {
-      chrome.tabs.sendMessage(tab.id, {method: 'editDeleteText'});
-    },
-  }
-});
+    'editor.contextDelete': {
+      title: 'editDeleteText',
+      type: 'normal',
+      contexts: ['editable'],
+      documentUrlPatterns: [URLS.ownOrigin + 'edit*'],
+      click: (info, tab) => {
+        chrome.tabs.sendMessage(tab.id, {
+          method: 'editDeleteText'
```

Comment:
This robotic expansion of literals doesn't make the code more readable. On the contrary, it's less readable in such cases as this one where only one parameter is used.